test/test_helper.exs

```diff
@@ -1 +1,4 @@
+Mox.defmock(Screenplay.RoutePatterns.Mock, for: Screenplay.RoutePatterns.Behaviour)
```

Comment:
I'm excited about this mock management library!!

Reply:
I think code will be much cleaner with it because we won't need any of the dependency injection that Screens uses. Much less parameters with default values. 🎉